tests/operators/test_plots.py

```diff
@@ -121,3 +121,25 @@ def test_plot_line_series_too_many_dimensions(cube, tmp_working_dir):
     """Error when cube has more than one dimension."""
     with pytest.raises(ValueError):
         plot.plot_line_series(cube)
+
+
+def test_plot_vertical_line_series(vertical_profile_cube, tmp_working_dir):
+    """Save a vertical line series plot."""
+    plot.plot_vertical_line_series(
+        vertical_profile_cube, series_coordinate="pressure", sequence_coordinate="time"
+    )
+    assert Path("untitled_473718.0.png").is_file()
+    assert Path("untitled_473721.0.png").is_file()
+
+
+def test_plot_vertical_line_series_no_series_coordinate(cube, tmp_working_dir):
+    """Error when cube is missing series coordinate (pressure)."""
+    cube = iris.cube.Cube([], var_name="nothing")
+    with pytest.raises(ValueError):
+        plot.plot_vertical_line_series(cube)
+
+
+def test_plot_vertical_line_series_too_many_dimensions(cube, tmp_working_dir):
+    """Error when cube has more than one dimension."""
+    with pytest.raises(ValueError):
+        plot.plot_vertical_line_series(cube)
```

Comment:
```suggestion
def test_plot_vertical_line_series_no_sequence_coordinate(vertical_profile_cube):
    """Error when cube is missing sequence coordinate (time)."""
    vertical_profile_cube.remove_coord("time")
    with pytest.raises(ValueError, match="Cube must have a time coordinate."):
        plot.plot_vertical_line_series(vertical_profile_cube)
```
Replace this test with one for the sequence coordinate missing.